include/aos/common/pkcs11utils.hpp

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright (C) 2023 Renesas Electronics Corporation.
+ * Copyright (C) 2023 EPAM Systems, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef AOS_IAM_MODULES_PKCS11_UTILS_HPP_
+#define AOS_IAM_MODULES_PKCS11_UTILS_HPP_
+
+#include "aos/common/pkcs11.hpp"
```

Comment:
Could we move utils inside `pkcs11.hpp`? Otherwise, create a separate folder: `pkcs11`: and put all pkcs11-related stuff there.